app/javascript/hooks/useTranslate.tsx

```diff
@@ -37,8 +37,8 @@ const initGoogleTranslate = () => {
 
 tWindow.initGoogleTranslate = initGoogleTranslate
 
-const useTranslate = (disable) => {
-  const languageInRoute = getCurrentLanguage()
+const useTranslate = (disable: boolean = true) => {
```

Comment:
if the feature flag value isn't ready (undefined), then we default `disable` to true